### ISSUE TITLE
use css module in the project

### ISSUE THREAD

**Issue by bhaskarSingh** [NJACKWinterOfCode/evolutions-webapp#5]
##### This issue is about {Issue Title}

- [yes ] I ensure that this issue is not a duplicate of any current open issues(s).
- [ yes] Assign this issue to me (I want to work on this).
- [ okay] I will send my PR only after I'm assigned to this issue by any maintainer.
- [yes ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by bhaskarSingh** [comment]
Rather than using global css which can get messy as the project grows(css class name over-riding issue leading unintentional bug) so we should move to using css modules & it's pretty easy to use too, we just need to add module.css extenion at the end of the css file & import it using destructing
